util/client/client.go

```diff
@@ -1,75 +1,111 @@
 package client
 
 import (
-	"context"
 	"fmt"
+	"os"
 	"time"
 
 	"github.com/pkg/errors"
-	"github.com/rancher/wrangler/pkg/clients"
-	"github.com/rancher/wrangler/pkg/schemes"
+	wranglerClients "github.com/rancher/wrangler/pkg/clients"
+	wranglerSchemes "github.com/rancher/wrangler/pkg/schemes"
+	"github.com/sirupsen/logrus"
 
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/informers"
-	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
 
-	corev1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Just noticed the previous is `corev1 "k8s.io/api/apps/v1"`, so is there any typo before?

Reply:
cc @c3y1huang is this caused by a wrong change by a previous format update?

Reply:
I think both 
"k8s.io/api/core/v1", "k8s.io/api/apps/v1" have `AddToScheme` function and they are actually pointing to the same place.
So both works fine, but in this PR, we need to use `corev1.NamespaceDefault` which  only in "k8s.io/api/core/v1" library

Reply:
It's different. One is for `app` workload related resource scheme but `core` is for basic resources like the pod below. 

[/vendor/k8s.io/api/core/v1/register.go#L36-L49](https://github.com/longhorn/longhorn-manager/blob/a6a15ae9f4546f796bd4ebdc31851536d11a8c8b/vendor/k8s.io/api/core/v1/register.go#L36-L49)
```go
var (
	// We only register manually written functions here. The registration of the
	// generated functions takes place in the generated files. The separation
	// makes the code compile even when the generated files are missing.
	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
	AddToScheme   = SchemeBuilder.AddToScheme
)

// Adds the list of known types to the given scheme.
func addKnownTypes(scheme *runtime.Scheme) error {
	scheme.AddKnownTypes(SchemeGroupVersion,
		&Pod{},
		&PodList{},
		&PodStatusResult{},
```

Reply:
> cc @c3y1huang is this caused by a wrong change by a previous format update?

It was previously [v1 k8s.io/api/apps/v1](https://github.com/longhorn/longhorn-manager/commit/c93aea9de7ed5e02feafcd70d3888d75742571ba#diff-7b3f4aabf038dc0bcb882dac26677c633b8cbb05ee4b57cfcbff61ea5051ec56L12).

Reply:
I see. Let's fix this if it has been backported.

Reply:
https://github.com/longhorn/longhorn-manager/pull/2238

Reply:
It's confusing, that we were using is core resource (secret below) but used the app scheme.

[/webhook/server/server.go#L126-L131](https://github.com/longhorn/longhorn-manager/blob/a6a15ae9f4546f796bd4ebdc31851536d11a8c8b/webhook/server/server.go#L126-L131)
```go
	client.Core.Secret().OnChange(s.context, "secrets", func(key string, secret *corev1.Secret) (*corev1.Secret, error) {
		if secret == nil || secret.Name != caName || secret.Namespace != s.namespace || len(secret.Data[corev1.TLSCertKey]) == 0 {
			return nil, nil
		}

		port := int32(types.DefaultWebhookServerPort)

```

Reply:
Fixed. Thank you.